src/main.rs

```diff
@@ -1,27 +1,37 @@
-use std::hash::Hash;
-use std::io::ErrorKind;
-use std::iter::FromIterator;
-use std::{
-    collections::HashSet,
-    fs::{self, File},
-    process::Command,
-};
-
-use anyhow::{bail, Error, Result};
+use std::path::Path;
+use std::{process::Command};
+use anyhow::{Error,Result,anyhow,bail};
 use clap::{Parser, Subcommand, ValueEnum};
 use glob::glob;
-use serde::{Deserialize, Serialize};
+use hocon::HoconLoader;
```

Comment:
[hocon](https://github.com/mockersf/hocon.rs) is unmaintained since January, we cannot rely on this crate.